### PR TITLE
Backport Tilt finalization from roda-sequel stack

### PR DESCRIPTION
**Pull request by fdr**
https://github.com/jeremyevans/roda-sequel-stack/commit/18cc6b1e2dce70dc113d8a79dc6e699fa773017a

Per usual, we cannot finalize Tilt in test as roda-sequel does, because rspec-mock relies on mutating classes at run-time.  But we ought to be able to do so in production (famous last words).

### CODE REVIEW

**Review by enescakir — APPROVED** [review]

File: config.ru [hunk 1]
```diff
@@ -5,3 +5,8 @@ require_relative "loader"
 clover_freeze
 
 run(Config.development? ? Unreloader : Clover.freeze.app)
+
+unless Config.development?
+  require "tilt/sass" unless File.exist?(File.expand_path("../compiled_assets.json", __FILE__))
+  Tilt.finalize!
+end
```
Comment (fdr) [inline config.ru:12]: @enescakir do we use sass at all? wondering if we should drop this branch entirely.

Comment (enescakir) [reply config.ru:12]: We don't use sass

Comment (fdr) [reply config.ru:12]: I'll delete tomorrow, then. Well, later today.

**Comment by fdr** [comment]
@enescakir decided to request a re-review from you, although perhaps this is hyperscrupulous; I simplified the sass branch

**Review by enescakir — APPROVED** [review]
I forgot to review this PR again

**Pull request merged**